### PR TITLE
fix annotations for columns with long data types

### PR DESCRIPTION
**Pull request by colbymelvin**
Addresses issue #621

### CODE REVIEW

File: spec/annotate/annotate_models_spec.rb [hunk 1]
```diff
@@ -958,6 +959,7 @@ def self.when_called_with(options = {})
         #  name(ＮＡＭＥ)       :string(50)       not null
         #  notes(ＮＯＴＥＳ)    :text(55)         not null
         #  no_comment           :text(20)         not null
+        #  location             :geometry_collect not null
```
Comment (colbymelvin) [inline spec/annotate/annotate_models_spec.rb]: This spec will fail with an `ArgumentError: negative argument` if you comment out the newly-added logic in `lib/annotate/annotate_models.rb`

File: spec/annotate/annotate_models_spec.rb [hunk 2]
```diff
@@ -943,7 +943,8 @@ def self.when_called_with(options = {})
         [:active,     :boolean, { limit: 1,  comment: 'ＡＣＴＩＶＥ' }],
         [:name,       :string,  { limit: 50, comment: 'ＮＡＭＥ' }],
         [:notes,      :text,    { limit: 55, comment: 'ＮＯＴＥＳ' }],
-        [:no_comment, :text,    { limit: 20, comment: nil }]
+        [:no_comment, :text,    { limit: 20, comment: nil }],
+        [:location,   :geometry_collection, { limit: nil, comment: nil }]
```
Comment (colbymelvin) [inline spec/annotate/annotate_models_spec.rb]: With this new column, this spec will fail with an ArgumentError: negative argument if you comment out the newly-added logic in lib/annotate/annotate_models.rb

**Comment by drwl** [comment]
@colbymelvin Thanks for reporting the issue + working on a fix. I'm trying to understand the root cause more, but the PR looks fairly straight forward. I'll try and carve out some time tonight or tomorrow to review.

**Comment by drwl** [comment]
Looks good to me. Thanks again for reporting the issue and submitting a PR.

**Comment by ilevin350** [comment]
Any idea when this fix will be cut into a production release of the gem?

**Pull request merged**